src/components/CarLoader.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Grid from '@mui/material/Grid';
+
+const LoaderScreen = () => (
+  <Grid container className="loader">
+    <h3 className="loader-heading">Finding perfect rides for you.</h3>
+    <img src="./loadingCar.gif" alt="loading-icon" className="loader-image" />
+  </Grid>
+);
+export default LoaderScreen;
```

Comment:
Let component and file names match

Reply:
sorry didnt notice, happened by mistake, updating it.